partner_event/__manifest__.py

```diff
@@ -4,17 +4,18 @@
 # Copyright 2016 Tecnativa S.L. - Vicent Cubells
 # Copyright 2017 Tecnativa S.L. - David Vidal
 # Copyright 2018 Jupical Technologies Pvt. Ltd. - Anil Kesariya
+# Copyright 2020 Tecnativa S.L. - Víctor Martínez
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 {
     "name": "Link partner to events",
-    "version": "12.0.1.0.1",
+    "version": "13.0.1.0.0",
     "category": "Marketing",
     "author": "Tecnativa," "Odoo Community Association (OCA)",
     "website": "https://www.tecnativa.com",
-    "development_status": "Production/Stable",
+    "development_status": "Alpha",
```

Comment:
Why are you changing this status? It should be the same. Check https://odoo-community.org/page/module-maturity-levels

Reply:
Sorry, I thought that when doing the migration you should change.
I write it down for a future that must be left.